src/main/java/dev/openfeature/javasdk/NoOpProvider.java

```diff
@@ -25,7 +25,7 @@ public ProviderEvaluation<Boolean> getBooleanEvaluation(String key, Boolean defa
         return ProviderEvaluation.<Boolean>builder()
                 .value(defaultValue)
                 .variant(PASSED_IN_DEFAULT)
-                .reason(Reason.DEFAULT)
+                .reason(Reason.DEFAULT.toString())
```

Comment:
I think it's still good to have an enum of default reasons. We just convert them to strings were applicable. We can instead use string constants... anyone have opinions on this?

Reply:
Yes this a good idea to keep the enum 👍